code/places-monitor-android/src/phone/java/com/adobe/marketing/mobile/PlacesLocationManager.java

```diff
@@ -89,6 +89,14 @@ void startMonitoring() {
 				PlacesActivity.askPermission(requestedLocationPermission);
 				return;
 			}
+		} else if(requestedLocationPermission == PlacesMonitorLocationPermission.NONE) {
+			// if the location permission hasn't been already granted, do not begin location tracking.
+			// log a message letting the developer know that the permission should be requested by the application.
+			if(!(PlacesActivity.isWhileInUsePermissionGranted() || PlacesActivity.isBackgroundPermissionGranted())) {
+				Log.warning(PlacesMonitorConstants.LOG_TAG, "Places Monitor doesn't have apps location permission to start monitoring. Please request for location permission" +
+						"in your application or call startMonitoring by setting setLocationPermission API to WHILE_USING_APP or ALWAYS_ALLOW");
```

Comment:
need a space either at the end of the string on 96 or at the beginning of the string on 97